src/ohri-form.component.test.tsx

```diff
@@ -232,6 +236,50 @@ describe('OHRI Forms:', () => {
     });
   });
 
+  describe('Labour and delivery validation', () => {
+    fit('should show error toast when the birth count does not match birth info count', async () => {
```

Comment:
Please rename these tests to something more generic. I believe the naming could be something about verifying obs group count

Reply:
Noted.